### PR TITLE
Update default values for sealed secrets in service command

### PR DESCRIPTION
**Pull request by chetan-rns**
**What type of PR is this?**

> /kind bug

**What does this PR do / why we need it**:

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: pkg/cmd/service/add.go [hunk 1]
```diff
@@ -74,8 +74,8 @@ func newCmdAdd(name, fullName string) *cobra.Command {
 	cmd.Flags().StringVar(&o.InternalRegistryHostname, "image-repo-internal-registry-hostname", "image-registry.openshift-image-registry.svc:5000", "Host-name for internal image registry e.g. docker-registry.default.svc.cluster.local:5000, used if you are pushing your images to the internal image registry")
 	cmd.Flags().StringVar(&o.PipelinesFolderPath, "pipelines-folder", ".", "Folder path to retrieve manifest, eg. /test where manifest exists at /test/pipelines.yaml")
 
-	cmd.Flags().StringVar(&o.SealedSecretsService.Namespace, "sealed-secrets-ns", "kube-system", "Namespace in which the Sealed Secrets operator is installed, automatically generated secrets are encrypted with this operator")
-	cmd.Flags().StringVar(&o.SealedSecretsService.Name, "sealed-secrets-svc", "sealed-secrets-controller", "Name of the Sealed Secrets services that encrypts secrets")
+	cmd.Flags().StringVar(&o.SealedSecretsService.Namespace, "sealed-secrets-ns", "cicd", "Namespace in which the Sealed Secrets operator is installed, automatically generated secrets are encrypted with this operator")
```
Comment (bigkevmcd) [inline pkg/cmd/service/add.go:77]: This was caused by us repeating the same logic over, we should simplify this to a simple function that adds any duplicated vars to a `*cobra.Command`

**Review by bigkevmcd — APPROVED** [review]

**Comment by bigkevmcd** [comment]
Let's fix this in a better way in a separate branch.

File: pkg/cmd/service/add.go (hunk 1 above)
Comment (wtam2018) [reply pkg/cmd/service/add.go:77]: @chetan-rns are we removing --sealed-secrets-svc flag?

Comment (wtam2018) [reply pkg/cmd/service/add.go:77]: There are constants `sealedSecretsController` and `sealedSecretsNS` in pkg/cmd/bootstrap and we are not using.

**Pull request merged**